pallets/api/src/extension.rs

```diff
@@ -136,17 +136,178 @@ fn version(env: &impl Environment) -> u8 {
 mod tests {
 	use frame_support::pallet_prelude::Weight;
 	use pop_chain_extension::Ext;
+	use sp_core::ConstU8;
 
-	use super::*;
+	use super::{DispatchError::*, *};
 	use crate::extension::Prepender;
 
+	#[test]
+	fn func_id_works() {
+		// Test ensuring little endian.
```

Comment:
Not so sure how necessary this comment is but I once made a spelling mistake using big endian which fails some of the tests and therefore added it.

Reply:
Looking at it again the second test might not be necessary as the first one was the one that failed anyway.